docs/developers/documentation/index.md

```diff
@@ -6,28 +6,69 @@ documentation.
+[creating and editing a new file](https://docs.github.com/en/repositories/working-with-files/managing-files/creating-new-files)
+or [editing an existing file](https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files)
+on the [napari-docs](https://github.com/napari/docs) GitHub repository.
+You can also first [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the [napari-docs](https://github.com/napari/docs) repository to your own GitHub account, create a [feature branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository#creating-a-branch), upload/create/edit files through the GitHub web interface, and then [open a pull request from your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) back to [napari-docs](https://github.com/napari/docs).
+
+You will also be able to [preview the documentation](use-ci-artifacts) as it would appear on [napari.org](https://napari.org/dev) by
+downloading the built documentation via a link provided by a comment from the `github-actions` bot, or previewing a live version of the website corresponding to your changes on [CircleCI](https://circleci.com/).
+A member of the maintenance
+team will help with updating the [napari.org](https://napari.org/dev) table of contents where necessary (by placing a reference to your new file in [docs/_toc.yml](update-toc)) and making sure your documentation has built correctly.
+
+
+(prerequisites)=
+## Prerequisites for a local setup to contribute to the napari documentation
 
 Prerequisites depend on the type of contribution you wish to make. In general,
 you will require:
 
-- Some familiarity with [`git`](https://git-scm.com);
+- Some familiarity with [`git`](https://git-scm.com).
 - A [GitHub](https://github.com) account.
 
+```{note}
+The napari documentation is built using `make` which does not work on paths which contain spaces.
+It is important that you clone the `napari/docs` repository to a path that does not contain spaces.
+For example, `C:\Users\myusername\Documents\GitHub\napari-docs` is a valid path, but \
+`C:\Users\my username\Documents\GitHub\napari-docs` is not.
+```
+
 You should first [fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
 and clone the [napari/napari](https://github.com/napari/napari) and the [napari/docs](https://github.com/napari/docs) repo to your
 machine. Next, navigate to your local clone of the `napari/docs` repository:
 
 ```bash
-git clone git@github.com:<your-username>/docs.git napari-docs
+git clone https://github.com/<your-username>/napari.git
+git clone https://github.com/<your-username>/docs.git napari-docs
```

Comment:
Github does not allow for authorization via username and password. So it may be hard to push to repositories created in this way, as it may need to set up token authorization with not-so-good documentation and less googlable errors.

Reply:
Yeah that's a good point thanks @Czaki, I've gotten so used to the token authentication that I forgot this can be a bit awkward.

Do you think the SSH setup steps are clearer? I.e. changing that to
```
git clone git@github.com:<your-username>/napari-docs.git
git clone git@github.com:<your-username>/docs.git napari-docs
```

Or would you recommend something else?

Reply:
I prefer to check the current state of github documentation (I do not have time for this today) and do as they suggest and put link to their documentation somewhere. It will reduce the maintenance burden. 

Based on my memory, they suggest ssh.

Reply:
It doesn't seem to recommend any particular option from what I can see in the [Cloning a repository guide](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository). However, there is a helpful [Troubleshooting cloning guide](https://docs.github.com/en/get-started/using-github/troubleshooting-connectivity-problems) which would probably good to to put in the docs.

Reply:
What about the `gh` command line tool? Is that a solution? it has some other upside like checking out PRs.